setup/upgrade_major.rst

```diff
@@ -120,6 +127,48 @@ done!
 
         $ SYMFONY_DEPRECATIONS_HELPER=max[total]=999999 php ./bin/phpunit
 
+Deprecations in logs
+~~~~~~~~~~~~~~~~~~~~
+If you use the Symfony default configuration, the deprecations are also logged in a ``prod.deprecations.log`` file (prod environment):
```

Comment:
iirc this config is not enabled now by default in the recipe

Reply:
Yes you're right, it's not enabled but present in the recipe as a comment because as a rule of thumb you only need to log deprecations in prod when you plan to migrate to the next major. IMHO it may be considered as the default configuration even if it's not always enabled.

Reply:
Understood, but maybe rewrote mentioning that this need to be enable, as by default config, it is not the case, and one can understood it is and then never find this file :)

Reply:
Oh yes sorry; I didn't make attention that the problem was in this section, thank you 👍

Reply:
Updated 🙂